### ISSUE TITLE
List text look different from the rest of the content

### ISSUE THREAD

**Issue by Hjdskes** [mtn/cocoa-eh-hugo-theme#13]
This picture shows how the text is smaller and has a different color.

![screenshot from 2017-02-25 18-23-29](https://cloud.githubusercontent.com/assets/1260727/23333184/8b7be584-fb87-11e6-9e31-bbd0934303aa.png)

I fixed this by adding the following to the css, but there might be a better way to fix this.

```css
section.main .content .markdown ul,
section.main .content .markdown ol,
section.main .content .markdown dl {
  margin-bottom: 1rem;
  color: #313537;
  font-size: 1.2em;
  letter-spacing: 0.01rem;
  font-family: 'Source Sans Pro', sans-serif;
  font-weight: 100;
  text-align: justify;
}
```

**Comment by fuegoio** [comment]
Hi !

This was because the style of text was only applied to the `p` tag, but now it is on the whole text in markdown (except titles and other special tags).

Fixed with 1866a82318f19f4ad10b48640c299e8ee7f0514a

**Comment by Hjdskes** [comment]
Thanks!
